pkgs/pkgs-lib/default.nix

```diff
@@ -0,0 +1,6 @@
+{ lib, pkgs }: {
```

Comment:
There should be a docstring or README.md describing the idea behind pkgs-lib.

Reply:
@Profpatsch Just added a docstring explaining pkgs-lib: https://github.com/NixOS/nixpkgs/pull/75584/files#diff-fa126a2182114f76a8e8df7ed256395d. Does that look good?